themes/bootstrap3/templates/record/save.phtml

```diff
@@ -7,7 +7,7 @@
     . '<li>' . $this->recordLink()->getBreadcrumb($this->driver) . '</li> '
     . '<li class="active">' . $this->transEsc('Save') . '</li>';
 ?>
-<h2><?=$this->transEsc("add_favorite_prefix") ?> <span class="title-in-heading"><?=$this->escapeHtml($this->driver->getBreadcrumb())?></span> <?=$this->transEsc("add_favorite_suffix") ?></h2>
+<h2><?=$this->translate("add_to_favorites_html", ['%%title%%' => '<em>' . $this->escapeHtml($this->driver->getBreadcrumb()) . '</em>']) ?></h2>
```

Comment:
Taken care of: https://github.com/vufind-org/vufind/pull/860/commits/2087e51393e5a2c71c305843b4de227d3de408f1